2019-03-11 11:32:35.669 [main] DEBUG com.isagod.service.impl.WechatAuthServiceImpl -             insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
### The error may involve com.isagod.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT   INTO   tb_wechat_auth(user_id,open_id,create_time)   VALUES   (?,?,?)
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
; SQL []; Duplicate entry 'dafahizhfdhaih' for key 'open_id'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
